mambaSharedFramework/Pantos-Generic Playlist Parsing/Pantos-Generic Tag Validators/PlaylistValidator.swift

```diff
@@ -29,11 +29,11 @@ public extension ExtensiblePlaylistValidator {
      
      - parameter playlist: A `PlaylistInterface` to validate
      
-     - returns: An array of `HLSValidationIssue`s. Will be empty if no issues are found.
+     - returns: An array of `ValidationIssue`s. Will be empty if no issues are found.
```

Comment:
`ValidationIssue` in swiftdoc doesn't match `PlaylistValidationIssue` in function definition below.